.rubocop.yml

```diff
@@ -6,3 +6,6 @@ Naming/HeredocDelimiterNaming:
 
 Lint/NestedMethodDefinition:
   Enabled: false
+
+Performance/HashEachMethods:
```

Comment:
Most objects constructed by openapi3_parser support .each but not .each_value but Rubocop treats it as a hash.

Reply:
Fair enough. Makes me wonder if we should add `each_value` to the parser at some point though (cc @kevindew)